style.css

```diff
@@ -101,3 +103,354 @@
+  margin: 0;
+  gap: 12px;
+  width: 236px;
+  height: 32px;
+}
+
+.content .clint {
+  padding: 8px 0;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  width: 55px;
+  height: 32px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.content .counter {
+  height: 8px;
+  width: 8px;
+  border-radius: 10px;
+  background: #c1c7d0;
+}
+
+.content .role {
+  height: 16px;
+  width: 106px;
+  top: 8px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 15px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+s.content .year {
+  height: 16px;
+  width: 29px;
+  top: 8px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  color: #7a869a;
+  line-height: 16px;
+  font-size: 15px;
+}
+
+.description {
+  font-family: Poppins, Arial, sans-serif;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  text-align: left;
+  color: #344563;
+  margin-top: 0;
+  margin-bottom: 0;
+  padding-left: 12px;
+}
+
+.card-work ul {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+.card-work ul li {
+  display: inline;
+}
+
+.card-work .main-buttons {
+  display: flex;
+  flex-direction: row;
+  gap: 8px;
+  width: 230px;
+  height: 34px;
+}
+
+.card-work .button {
+  padding: 12px 12px;
+  height: 24px;
+  display: flex;
+  align-items: center;
+  background: #ebebff;
+  border-radius: 8px;
+  margin-right: 4px;
+  text-align: left;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+.card-work .buttons {
+  width: 295px;
+  height: 72px;
+  display: flex;
+  align-items: center;
+}
+
+.card-work .btn {
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+  justify-content: center;
+  padding: 12px;
+  gap: 10px;
+  width: 124px;
+  height: 48px;
+  line-height: 24px;
+  border-radius: 8px;
+}
+
+.btn-project {
+  font-family: Poppins, Arial, sans-serif;
+  font-size: 14px;
+  font-weight: 500;
+  line-height: 24px;
+  text-align: center;
+  display: flex;
+  align-items: center;
+  letter-spacing: 0.03em;
+  margin-right: 180px;
+}
+
+.card-work .btn-project {
+  border: 1px solid #6070ff;
+  color: #396df2;
+  background-color: #fff;
+  text-align: center;
+  display: flex;
+  align-items: flex-start;
+}
+
+.card-work .btn-project:hover {
+  background: #6070ff;
+  color: #fff;
+}
+
+.card-work .btn-project:active {
+  background: #2230d2;
+  color: #fff;
+}
+
+.card-work .btn-project:disabled {
+  border: 1px solid #c1c7d0;
+  color: #5e6c84;
+}
+
+.about-myself {
+  background-color: white;
+  border-top-right-radius: 80px;
+  width: 375px;
+  margin: -20px;
+  padding: 114px 24px;
+}
+
+.first .intro {
+  height: 101px;
+  width: 327px;
+  font-family: Poppins, Arial, sans-serif;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  text-align: left;
+}
```

Comment:
- [ ] [REQUIRED] Please make changes!

- [REMEMBER]  Use `font-family: 'Poppins'` of all text provided in the project!

- 1-The font thickness and layout in the text about yourself are not like [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122), because your text part is not visible in some mobile size conscious part.
- 2-Please change the color of the text to `#7F8CFF`.
- 3- This text design is not the same as in Figma design, edit them using `font-style: normal; font-weight: 500; font-size: 20px;`

![image](https://user-images.githubusercontent.com/63915024/170842295-c8b0450d-b9e0-4400-a625-588c9d382360.png)